enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java

```diff
@@ -1,46 +1,35 @@
 package cuchaz.enigma.translation.mapping.serde.recaf;
 
-import com.google.common.collect.Sets;
-import com.google.common.jimfs.Jimfs;
 import cuchaz.enigma.ProgressListener;
 import cuchaz.enigma.translation.mapping.EntryMapping;
 import cuchaz.enigma.translation.mapping.tree.EntryTree;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.FileSystem;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.HashSet;
 import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class TestRecaf {
 
     @Test
     public void testIntegrity() throws Exception {
-        Set<String> contents;
-        try (InputStream in = getClass().getResourceAsStream("/recaf.mappings")) {
-            contents = Sets.newHashSet(new String(in.readAllBytes(), StandardCharsets.UTF_8).split("\\R"));
-        }
+        Path path = Path.of("src/test/resources/recaf.mappings");
+        Set<String> contents = new HashSet<>(Files.readAllLines(path));
```

Comment:
This should be the same as before, we want to use the file from the same jar as the current class

Reply:
I readded jimfs